src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -56,12 +59,22 @@ internal class GrpcWorker : IWorker
             _startupOptions = startupOptions ?? throw new ArgumentNullException(nameof(startupOptions));
             _serializer = workerOptions.Value.Serializer ?? throw new InvalidOperationException(nameof(workerOptions.Value.Serializer));
             _inputConversionFeatureProvider = inputConversionFeatureProvider ?? throw new ArgumentNullException(nameof(inputConversionFeatureProvider));
+            _appLocation = Environment.GetEnvironmentVariable("AzureWebJobsScriptRoot")!;
```

Comment:
This was the easiest way I could think of to get the path I needed to find and read from functions.metadata -- I wasn't able to find it in any of the objects passed in (double checked by debugging here in VS). In previous iterations, this info was provided from the host gRPC message, but we want to gather the function metadata asap before that message since we know we'll need it anyways.